.github/workflows/test-suite-desktop-e2e.yml

```
@@ -36,24 +36,39 @@ jobs:
-          - TEST_GROUP: "@group=wallet"
-            CONTAINERS: "trezor-user-env-unix bitcoin-regtest"
-
     steps:
-        # Electron requires unprivileged user namespaces to function properly. 
-        # Disabling this security rule allows Electron to create sandboxed processes
-        # without requiring elevated privileges, which is essential for running the application.
-        # This is workaround until electron builder solves this issue in future release.
+      - name: Debug Previous Runs
+        env:
+          GITHUB_TOKEN: ${{ secrets.TREZOR_BOT_TOKEN }}
+        run: |
```

actionlint (1.7.4)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions for more details

(expression)